utils.py

```diff
@@ -165,7 +166,8 @@ def setup_resnet(model, device, num_devices, batch_size, address, cluster=True):
                                        model,
                                        "TORCH",
                                        device,
-                                       batch_size)
+                                       batch_size,
+                                       min_batch_size=batch_size)
```

Comment:
why is there a batch_size var and a min batch size var, with the batch size assigned to min batch size? can this var be taken away in the smartsim api?